code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -23,7 +23,7 @@
 		var/fillevel = gulp_size
 
 		if(!R.total_volume || !R)
-			user << "\red None of [src] left, oh no!"
+			user << "<span class = 'warning'> None of [src] left, oh no!</span>"

```

Comment:
`<span class='warning'>` instead of `<span class = 'warning'>`, the spaces matter if you're trying to grep or mass-replace something, so everything must use no spaces to be consistent with span classes that are already present.